datadog_checks_dev/datadog_checks/dev/fixtures/plugin.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+try:
+    from datadog_checks.stubs import aggregator as __aggregator
+
+    @pytest.fixture
+    def aggregator():
+        __aggregator.reset()
+        return __aggregator
+
+except ImportError:
+    @pytest.fixture
+    def aggregator():
+        return 'datadog_checks_base is not installed!'
```

Comment:
I think this should just raise, returning a string opens to a set of problems downstream